app/controllers/v1/songs_controller.rb

```diff
@@ -1,7 +1,17 @@
 class V1::SongsController < ApplicationController
+
+
```

Comment:
Don't want these two extra blank lines, or the one at the bottom before the last `end`